### PR TITLE
Review draft PR creation

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

**Comment by annevk** [comment]
I guess the problem with makeGroupsOfFive is that if we add a standard while the current process is still ongoing, it might cause another standard to be published again a month later. So maybe @foolip's approach of hardcoding months (see https://github.com/whatwg/sg/issues/127) in db.json is the way to go.

**Comment by domenic** [comment]
+1 for hardcoding months; I like the predictability, and it might even be information we want to surface somewhere someday (e.g. on https://whatwg.org/workstreams)

**Comment by annevk** [comment]
Months are part of db.json now. The way I was thinking of taking this further in July:

* No need for arguments, just get the current month.
* If the current day is far from the third Monday of the month, maybe log a warning.
* Get all the relevant standards for the month and create PRs for them.
* Optionally, migrate more of `make preview` into this script rather than have that spread out across our various standards.

**Comment by annevk** [comment]
@foolip (and @domenic) this is now updated to take into account the updated db.json. We could try it out on Monday. Suggestions and such welcome as well as thoughts on the "Optionally" above.

File: review.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import argparse, datetime, json, os, subprocess, requests
+
+
+def is_third_monday(d):
+    return d.weekday() == 0 and 15 <= d.day <= 21
+
```
Comment (foolip) [inline review.py:8]: Nit: another newline here? (The amount of space between things is inconsistent in other places too.)

File: review.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        maybe_create_pr(shortname)
+        os.chdir(".")
+
+def maybe_create_pr(shortname):
+    subprocess.run(["git", "checkout", "master"], capture_output=True)
+    commits = subprocess.run(["git", "log", "--pretty=oneline", "--abbrev-commit", "--abbrev=8", "--max-count=40"], capture_output=True).stdout
+    for commit in commits.split(b"\n"):
+        # Remove the leading prefix (see --abbrev=8 above, which cannot be 0, plus a space)
+        commitTitle = commit[8+1:]
+        if commitTitle.startswith(b"Meta:"):
```
Comment (foolip) [inline review.py:57]: What about "Editorial:"? Or is it the message below that needs to change?

File: review.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+def maybe_create_prs(shortnames):
+    for shortname in shortnames:
+        os.chdir("../{}".format(shortname))
+        maybe_create_pr(shortname)
+        os.chdir(".")
+
+def maybe_create_pr(shortname):
+    subprocess.run(["git", "checkout", "master"], capture_output=True)
+    commits = subprocess.run(["git", "log", "--pretty=oneline", "--abbrev-commit", "--abbrev=8", "--max-count=40"], capture_output=True).stdout
```
Comment (foolip) [inline review.py:53]:
```suggestion
    commits = subprocess.run(["git", "log", "--format=%s", "--max-count=40"], capture_output=True).stdout
```

Printing only the subject line for each commit would simplify this loop a bit overall.

File: review.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    for commit in commits.split(b"\n"):
+        # Remove the leading prefix (see --abbrev=8 above, which cannot be 0, plus a space)
+        commitTitle = commit[8+1:]
```
Comment (foolip) [inline review.py:56]: I'd suggest calling this something with subject to match Git's terminology.

File: review.py [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--force", dest="force", action="store_true")
+    args = parser.parse_args()
+
+    today = datetime.datetime.today()
+    ideal_publication_diff = days_from_third_monday(today)
+
+    if ideal_publication_diff == 0:
+        print("Right on, today is the day!")
```
Comment (foolip) [inline review.py:89]: I love it!

**Pull request merged**